extension/js/popup.js

```diff
@@ -11,7 +11,7 @@ function getOnboardingPanels() {
     "panel2": {
       "imgSrc": "tip2-icon.svg",
       "tipHeadline": "Don't see the icon?",
-      "tipBody":"Right click (Windows) or control+click (Mac) to open the context menu easily generate a new relay from there",
+      "tipBody":"Right-click (Windows) or Control-click (macOS) to open the context menu easily generate a new relay from there.",
```

Comment:
nit: This feels odd here since we've been removing instances of "relay" elsewhere above.

Reply:
I will bring this up with UX

Reply:
Also says "open the context menu easily generate" when it should say "open the context menu *to* easily generate" ?

Reply:
Or should it be "..._to_ open the context menu _and_ easily generate a new..."